setupdo.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
`setupdo` all run together isn't easy to mentally parse. I'd vote instead for something like `setup_droplet.sh`?